cms/templates/blog.html

```diff
@@ -0,0 +1,56 @@
+{% extends "base.html" %}
+
+{% load render_bundle image_version_url %}
+
+{% block title %}{{ page.title }}{% endblock %}
+
+{% block headercontent %}
+  <div id="header"></div>
+  {% render_bundle 'header' %}
+{% endblock %}
+
+{% block content %}
+<div class="blog-page">
+  {% if page.banner_image %}
```

Comment:
In the case of a background from Wagtail it doesn't scale the image height, Keeps it at 188px.

<img width="1792" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/81ccca32-2ea8-4913-aacf-39893dbdd2b7">